docs/submission/validating-molecular-data.md

```diff
@@ -0,0 +1,442 @@
+		"submitter_sequencing_experiment_id": "DASH-SE-1.2",
+		"experimental_strategy": "WGS",
+		"sequencing_center": "",
+		"platform": "ILLUMINA",
+		"platform_model": null,
+		"sequencing_date": null
+	}
+}
+```
+
+
+</TabItem>
+</Tabs>
+
+<!---  Tabs end here -->
+
+See github for the same example of a correctly formatted payload from a [normal sample](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/example_payloads/dash1_normal.json) and a [tumour sample](https://github.com/icgc-argo/argo-metadata-schemas/blob/master/example_payloads/dash1_tumour.json).
+
+Depending on the type of sequencing that was done, the `read_groups` section of the payload will need to be adjusted. Examples of correctly formatted `read groups` that meet [the rules](/docs/submission/submitting-molecular-data#file-and-data-validation-rules) for paired-end vs single-end sequencing are are shown here as examples. 
+
+<!---  Tabs start here -->
+
+<Tabs
+  defaultValue="BAM-paired"
+  values={[
+    { label: 'BAM (paired)', value: 'BAM-paired', },
+    { label: 'BAM (single)', value: 'BAM-single', },
+    { label: 'FASTQ (paired)', value: 'FASTQ-paired', },
+    { label: 'FASTQ (single)', value: 'FASTQ-single', },
+  ]
+}>
+<TabItem value="BAM-paired">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+  	"read_group_id_in_bam": "C0HVY.2",
+  	"submitter_read_group_id": "C0HVY.2",
+  	"platform_unit": "74_8a",
+  	"is_paired_end": true,
+  	"file_r1": "test_rg3.bam",
+  	"file_r2": "test_rg4.bam",
+  	"read_length_r1": 150,
+  	"read_length_r2": 16,
+  	"insert_size": 232,
+  	"sample_barcode": null,
+  	"library_name": "Pond-147579"
+  }
+```
+
+</TabItem>
+<TabItem value="BAM-single">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": "C0HVY.2", 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8a",
+    "is_paired_end": true,
+    "file_r1": "test_rg55.bam",
+    "file_r2": null,
+    "read_length_r1": 150,
+    "read_length_r2": null,
+    "insert_size": 232,
+    "sample_barcode": null,
+    "library_name": "Pond-147579"
+  }
+```
+
+</TabItem>
+<TabItem value="FASTQ-paired">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": null, 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8b",
+    "is_paired_end": true,
+    "file_r1": "test_rg23.fq",
+    "file_r2": "test_rg24.fq",
+    "read_length_r1": 150,
+    "read_length_r2": 125, 
+    "insert_size": 298,
+    "sample_barcode": null,
+    "library_name": "Pond-147580"
+  }
+```
+
+</TabItem>
+<TabItem value="FASTQ-single">
+
+```json
+  "read_group_count": 1,
+  "read_groups": {
+    "read_group_id_in_bam": null, 
+    "submitter_read_group_id": "C0HVY.2",
+    "platform_unit": "74_8b",
+    "is_paired_end": false,
+    "file_r1": "test_rg5.fq",
+    "file_r2": null,
+    "read_length_r1": 150,
+    "read_length_r2": null, 
+    "insert_size": 298,
+    "sample_barcode": null,
```

Comment:
perhaps it's more helpful not to show an example full of "null" values, so they get a sense of what can go in there

Reply:
no this is important that they are null! for the different read group sections ,based on the pairing status different things can or should be null

Reply:
how do they now when they aren't null and what they put in there if not null?

Reply:
i moved the rules closer (right below the example). 

its in the tabs  - BAM v FASTQ, and paired v single